public/js/showoff.js

```diff
@@ -304,6 +304,7 @@ function setupMenu() {
         .text(slidePath)
         .append(icon)
         .click(function() {
+          $(this).children('i').toggleClass('fa-angle-down fa-angle-up');
```

Comment:
the issue here is that some child elements start expanded and the toggle doesn't validate what state they should be in